public/components/TagEdit/formComponents/TagAdBlocking.react.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+
+export default class TagAdBlocking extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  setAdBlockingLevel(e) {
+    this.props.updateTag(
+      Object.assign({}, this.props.tag, {
+        adBlockingLevel: e.target.value
+      })
+    );
+  }
+
+  setContributionBlockingLevel(e) {
+    this.props.updateTag(
+      Object.assign({}, this.props.tag, {
+        contributionBlockingLevel: e.target.value
+      })
+    );
+  }
+
+  render() {
+    const blockingLevels = ["None", "Suggest", "Force"];
```

Comment:
Worth exporting these to DRY [this](https://github.com/guardian/tagmanager/pull/359/files#diff-ed9dd6f83909e3393f3af34d2a20747dR16) out a bit?

Reply:
Also, the string values here are different to the string values in `BlockingLevel` which are all upper case.